src/AppBundle/Twig/WikiExtension.php

```diff
@@ -69,6 +70,24 @@ public function userLink($username, $projectUrl, $label = null)
     }
 
     /**
+     * Get a link to the given user's userpage, or to Special:Contribs if $username is an IP
+     * @param  string $username   Username
+     * @param  string $projectUrl Project domain and protocol such as https://en.wikipedia.org
+     * @param  string [$label]    The link text, defaults to $username
+     * @return string Markup
+     */
+    public function groupLink($group, $projectUrl, $label = null)
+    {
+        if (!$label) {
+            $label = $group;
+        }
+        // Ignoring this inspection, as we want all of the output on one line.
+        // @codingStandardsIgnoreStart
+        return "<a href='$projectUrl/wiki/index.php?title=Special:ListUsers&group=$group&creationSort=1&limit=50' target='_blank'>$label</a>";
```

Comment:
We shouldn't assume the path to be `/wiki/` (although, I'm not sure how we're handling this; it's a common thing).